euth/dashboard/static/language_switch/react_language_switch.jsx

```diff
@@ -70,10 +66,29 @@ class LanguageSwitch extends React.Component {
           </button>
           <ul className="dropdown-menu">
             {
-              this.props.languages.map(languageCode => {
+              this.props.languages.map((languageCode, i) => {
+                return (
+                  <span key={languageCode}>
```

Comment:
@phillimorland I am a bit unhappy with returning an empty span for the languages that shouldn't be in the list, but couldn't think of a better way. (Apart from also having a list of the inactive languages in the state, but that would be a bit annoying to always update two lists, then.)

Reply:
i think this is ok for now to be honest